EKF/ekf.h

```diff
@@ -256,7 +256,7 @@ class Ekf : public EstimatorInterface
 	void get_ekf_soln_status(uint16_t *status);
 
 	// return the quaternion defining the rotation from the EKF to the External Vision reference frame
```

Comment:
// return the quaternion defining the rotation from the External Vision to the EKF reference frame